lib/web/desktop.go

```diff
@@ -98,6 +103,12 @@ func (h *Handler) createDesktopConnection(
 		return trace.BadParameter("height missing or invalid")
 	}
 
+	if width > maxRDPScreenWidth || height > maxRDPScreenHeight {
+		return trace.BadParameter("screen size of %d x %d is greater than the maximum allowed by RDP (%d x %d)",
+			width, height, maxRDPScreenWidth, maxRDPScreenHeight,
+		)
+	}
```

Comment:
Erroring here causes the UI to hang at the spinner, though that's a more general problem with the error handling all along this codepath. Possibly related with https://github.com/gravitational/teleport/issues/10595

Reply:
That's super annoying, but not new. Let's get a fix for that out separately (but asap)